prime-router/src/main/resources/db/migration/V13__add_site_of_care_to_test_result_table.sql

```diff
@@ -0,0 +1,13 @@
+/*
+This SQL creates the tables of the DB. The Flyway tool applies this migration to create the database
+Follow this style guide https://about.gitlab.com/handbook/business-ops/data-team/platform/sql-style-guide/
+use VARCHAR(63) for names in organization and schema
+Copy a version of this comment into the next migration
+*/
+
+
+/*
+ * This SQL adds 'site_of_care' to the covid_result_metadata table
+ */
+
+ALTER TABLE covid_result_metadata ADD COLUMN site_of_care VARCHAR(128);
```

Comment:
Does it make sense to add this too?
```sql
alter table covid_result_metadata
    add constraint chk_crm_site_of_care
    check (length("site_of_care") > 0);
```
Do we expect empty values here? If so, do we want to normalize them to `null` (and thus make this field nullable?
If we make the field nullable, you can change the constraint to
```sql
alter table covid_result_metadata
    add constraint chk_crm_site_of_care
    check ("site_of_care" is null or length("site_of_care") > 0);
```

Reply:
I originally set it to a default of empty string but then removed it since I didn't see any others created that way. I'm not sure what the preference is for the metadata table, null or non empty?

Reply:
@MauriceReeves-usds Do you want me to add the check constraint above and make the column nullable?